src/vtbackend/ViCommands.cpp

```diff
@@ -1033,7 +1034,7 @@ CellLocation ViCommands::translateToCellLocation(ViMotion motion, unsigned count
                 --count;
             }
 
-            return result;
+            return { result.line, result.column - 1 };
```

Comment:
Hmm... While fixing it, would it make sense to add a test for it to `ViCommands_test.cpp`?

Is it possible that `result.column` could be `0` already and then subtracting `1` might be an issue?

Reply:
I will try to add a test for it, and also i figured out that we need this change only when we yank, so if previous command was not yank we should not go back by one character

Reply:
yeah, this is the thing with inclusive vs exclusive motions :)

Reply:
Now it is fixed